src/guides/protocol.md

```diff
@@ -0,0 +1,230 @@
+| (undefined name)                 |              | `CLOSED`          |
+| < 37                             | ~6 hours     | `OPEN`            |
+| < 720                            | ~5 days      | `BIDDING`         |
+| < 1,440                          | ~10 days     | `REVEAL`          |
+| > 0                              |              | `CLOSED`          |
+
+When reserved names are `CLAIM`ed on chain, they enter a
+[lockup period of about 30 days](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L243).
+During the `LOCKED` phase,
+[a reserved name can be re-claimed](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/blockchain/chain.js#L925-L936).
+This is to provide solutions to legacy DNS name holders whose keys are compromised.
+
+### Claims
+
+Legacy DNS names in the Alexa top 100,000 are reserved for their current owners.
+You can see the list in [names.json](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/covenants/names.json).
+
+[Learn more about the claims list.](https://github.com/handshake-org/hs-names)
+
+[Learn about on-chain name claims.](https://handshake-org.github.io/guides/claims.html)
+
+### Name rollout
+
+Un-reserved names are not all available immediately when mainnet launches. They
+become available over the first year, based on a simple algorithm that
+[compares the name's hash against a 52-week modulus.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/covenants/rules.js#L274-L295)
+
+The entire Handshake auction system is not activated until
+[the first ten days](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/protocol/networks.js#L229)
+of network life.
+
+### Airdrop
+
+Two thirds of the total currency supply is reserved for hundreds of thousands of
+Internet citizens in the open-source software community. Public keys were collected
+in various ways and added to a
+[merkle tree](https://github.com/handshake-org/hs-tree-data),
+whose root is
+[committed](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/primitives/airdropproof.js#L22)
+in consensus code. Airdrops are redeemed by submitting a proof to the network.
+
+Note that airdrop outputs ARE NOT included in the genesis block like other cryptocurrency
+airdrops. A Handshake airdrop is like a transaction that can be submitted at any time. In
+fact, because the faucet recipients are recorded by address and not public key,
+no signature is required to generate their airdrop transactions (of course a signature
+is required to SPEND from the airdrop, like any other address). Because these transactions
+can be generated and submitted by any node at any time, their
+[fees are hard-coded.](https://github.com/handshake-org/hsd/blob/b32c27bbb05e69b86c0e80aea3ecc394c5589186/lib/primitives/airdropproof.js#L18-L19)
```

Comment:
fees are hard-coded and enforced by consensus

Reply:
👍